portfolio/src/App.js

```
@@ -0,0 +1,53 @@
+
+function App() {
+  // Estado para controlar qual página mostrar
+  const [currentPage, setCurrentPage] = useState('home');
+
+  // Função para renderizar a página atual
+  const renderPage = () => {
+    switch(currentPage) {
+      case 'cv':
+        return <CV />;
+      case 'home':
+      default:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)